### ISSUE TITLE
Continuous Testing on Python 2.7

### ISSUE THREAD

**Issue by andreafrancia** [andreafrancia/trash-cli#313]
**Describe the bug**
Missing test workflow  for python 2.7. In the past trash-cli supported both python 2.7 and the newer versions of python 3.x leveraging the Git Hub Actions support. Recently Git Hub Action dropped the support of python 2.7 and so we no longer have a way to test the code automatically on Python 2.7.

**To Reproduce**
* Fork the project in your GitHub
* Enable Git Hub Actions on your repository
* Launch the actions and verify it no longer execute on python 2.7
* Edit `.github/workflows/run-tests.yml` and add '2.7' to the python versions 
  *  e.g.`python-version: ['2.7', '3.7', '3.8', '3.9', '3.10', '3.11']`
* Save, commit and push and see that is failing because 2.7 is not supported

**Expected behavior**
That have a way to launch tests on every commit on python 2.7 like the old days.

Probably is just a matter of running an action on a custom image based on an LTS distribution that still provide support for python 2.7. (For me) it is not an easy task.

I hope that someone skilled can contribute a solution.

**Comment by andreafrancia** [comment]
If I find a potentially dangerous bug I wish to be able to distribute the fix also to the people that are stuck with a legacy distributions.

**Comment by darix** [comment]
on the other hand more and more distros are in the process of actually dropping python2  and with that dropping six.
so with python 2 now really officially being EOL. it is the time to make trash-cli python3 only?

**Comment by a-detiste** [comment]
Yes distro would be happy https://wiki.debian.org/Python3-six-removal

**Comment by a-detiste** [comment]
I tried to patch-out six, but failed because I don't know the pecularities of old-style classes...
https://stackoverflow.com/questions/54867/what-is-the-difference-between-old-style-and-new-style-classes-in-python

**Comment by a-detiste** [comment]
Here is my patch

https://salsa.debian.org/debian/trash-cli/-/blob/master/debian/patches/remove_six.patch?ref_type=heads

**Comment by darix** [comment]
you marked this as done but i dont see any related commit in git?

**Comment by jvoisin** [comment]
@andreafrancia would you be interested in a pull-request dropping `six` from trash-cli?
